### PR TITLE
Raspios/buster 2.0.2-2

### PR DESCRIPTION
**Pull request by zhan-kunbus**
remove the depend of net-tools

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -14,7 +14,6 @@ Depends: ${shlibs:Depends},
  parted,
  e2fsprogs,
  libraspberrypi-bin,
- net-tools,
```
Comment (iluminat23) [inline debian/control:17]: You should add iproute2 as dependency. Yes iproute2 is in the default installation of debian, but it should be in the dependency of this package.

**Review by iluminat23 — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (zhan-kunbus) [reply debian/control:17]: good catch, I will add it.
but it's still not clear to me how the /usr/bin/ip get installed.
```
pi@RevPi2222:~ $ dpkg -L iproute2 | grep -w ip | grep -v ip-
/bin/ip
/usr/share/man/man8/ip.8.gz
/sbin/ip
pi@RevPi2222:~ $ which ip
/usr/sbin/ip
pi@RevPi2222:~ $ whereis ip
ip: /usr/bin/ip /usr/sbin/ip /usr/share/man/man7/ip.7.gz /usr/share/man/man8/ip.8.gz
pi@RevPi2222:~ $ ls -l /usr/bin/ip
-rwxr-xr-x 1 root root 503112 Dec  3  2020 /usr/bin/ip
pi@RevPi2222:~ $ ls -l /usr/sbin/ip
lrwxrwxrwx 1 root root 7 Dec  3  2020 /usr/sbin/ip -> /bin/ip
pi@RevPi2222:~ $ ls -l /sbin/ip
lrwxrwxrwx 1 root root 7 Dec  3  2020 /sbin/ip -> /bin/ip
pi@RevPi2222:~ $ ls -l /bin/ip
-rwxr-xr-x 1 root root 503112 Dec  3  2020 /bin/ip
```

Comment (iluminat23) [reply debian/control:17]: The iproute2 package installs it to `/bin/ip`. /bin is a symlink to `/usr/bin`.

**Comment by iluminat23** [comment]
Please squash the net-tolls remove and the iproute2 add commits. as in fact you replace the dependency.

File: debian/control (hunk 1 above)
Comment (zhan-kunbus) [reply debian/control:17]: good to know, thanks

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+piserial (2.0.2-2) buster; urgency=medium
+
+  * Remove the Depends of net-tools
```
Comment (iluminat23) [inline debian/changelog:3]: Recreate the changelog with the " Replace depend of net-tools with iproute2 " commit.

**Review by iluminat23 — CHANGES_REQUESTED** [review]

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**